pkg/oc/admin/diagnostics/diagnostics/cluster/aggregated_logging/diagnostic.go

```diff
@@ -153,20 +150,30 @@ func (d *AggregatedLogging) Requirements() (client bool, host bool) {
 }
 
 func (d *AggregatedLogging) Complete(logger *log.Logger) error {
-	if len(d.MasterConfigFile) > 0 {
-		var err error
-		d.masterConfig, err = hostdiag.GetMasterConfig(d.MasterConfigFile, logger)
+	if len(d.Project) > 0 {
+		return nil
+	}
+
+	// Check if any of the default logging projects are present in the cluster
+	for _, project := range defaultLoggingProjects {
+		d.Debug("AGL0030", fmt.Sprintf("Trying default logging project %q", project))
+		_, err := d.ProjectClient.Projects().Get(project, metav1.GetOptions{})
 		if err != nil {
-			return err
+			if kerrors.IsNotFound(err) {
+				d.Debug("AGL0031", fmt.Sprintf("Project %q not found", project))
+				continue
+			}
+			return fmt.Errorf("Failed fetching one of the default logging projects %q: %v", project, err)
```

Comment:
General convention is user returned errors start with lower case [fmt.Errorf("failed...) instead of fmt.Errorf("Failed...)] as these will be appended but this was not strictly enforced in our diags code, so i won't block on this.

Reply:
fixed both, thanks for your help, greatly appreciated!